### PR TITLE
Civ 3607 total amount you are claiming

### PR DESCRIPTION
**Pull request by GorilaNaranja**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-3607

### Change description ###
Total amount you’re claiming

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/views/features/eligibility/total-amount.njk [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <form method="post" novalidate>
+            {{ csrfProtection(csrf) }}
+            {{ govukRadios({
+              idPrefix: "totalAmount",
+              name: "totalAmount",
+              items: [
+                {
+                  value: 'over-25000',
```
Comment (TatsyMaslova) [inline src/main/views/features/eligibility/total-amount.njk:35]: Can you pass your enum totalAmountOptions in the view so you check against the enum values? this way if we do have to change the values, we only have to change it once in the enum

File: src/main/common/models/eligibility/totalAmountOptions.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export enum TotalAmountOptions {
+  OVER_25000 = 'over-25000',
+  LESS_25000 = 'less-25000',
+  UNKNOW = 'unknow',
```
Comment (laperla11) [inline src/main/common/models/eligibility/totalAmountOptions.ts:4]: typo; UNKNOWN = 'unknown` ,

File: src/main/views/features/eligibility/total-amount.njk [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                  text: t('PAGES.ELIGIBILITY_TOTAL_AMOUNT.OVER_25000'),
+                  checked: form.model.option === 'over-25000'
+                },
+                {
+                  value: 'less-25000',
+                  text: t('PAGES.ELIGIBILITY_TOTAL_AMOUNT.25000_OR_LESS'),
+                  checked: form.model.option === 'less-25000'
+                },
+                {
+                  value: 'unknow',
```
Comment (laperla11) [inline src/main/views/features/eligibility/total-amount.njk:45]: same typo here

File: src/test/unit/routes/features/eligibility/totalAmountController.test.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      await request(app).post(ELIGIBILITY_CLAIM_VALUE_URL).expect((res) => {
+        expect(res.status).toBe(200);
+        expect(res.text).toContain('There was a problem');
+      });
+    });
+
+    it('should render not eligible page when radio over-25000 is selected', async () => {
+      await request(app).post(ELIGIBILITY_CLAIM_VALUE_URL).send({ 'totalAmount': 'over-25000' }).expect((res) => {
+        expect(res.status).toBe(302);
+        expect(res.header.location).toBe(NOT_ELIGIBLE_FOR_THIS_SERVICE_URL + '?reason=claim-value-over-25000');
```
Comment (laperla11) [inline src/test/unit/routes/features/eligibility/totalAmountController.test.ts:43]: same function here

**Review by laperla11 — CHANGES_REQUESTED** [review]

**Review by TatsyMaslova — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]

File: src/main/views/features/eligibility/total-amount.njk (hunk 1 above)
Comment (GorilaNaranja) [reply src/main/views/features/eligibility/total-amount.njk]: Done!

File: src/main/common/models/eligibility/totalAmountOptions.ts (hunk 2 above)
Comment (GorilaNaranja) [reply src/main/common/models/eligibility/totalAmountOptions.ts]: Done!

File: src/main/views/features/eligibility/total-amount.njk (hunk 3 above)
Comment (GorilaNaranja) [reply src/main/views/features/eligibility/total-amount.njk]: Done!

File: src/main/routes/features/eligibility/totalAmountController.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  await form.validate();
+  if (form.hasErrors()) {
+    res.render(totalAmountViewPath, { form });
+  } else {
+    const cookie = req.cookies['eligibility'] ? req.cookies['eligibility'] : {};
+    cookie.totalAmount = req.body.totalAmount;
+    res.cookie('eligibility', cookie);
+    switch (totalAmount.option) {
+      case TotalAmountOptions.OVER_25000:
+        res.redirect(NOT_ELIGIBLE_FOR_THIS_SERVICE_URL + '?reason=claim-value-over-25000');
```
[the remark this answers is not in the document]
Comment (GorilaNaranja) [reply src/main/routes/features/eligibility/totalAmountController.ts]: Done!

File: src/test/unit/routes/features/eligibility/totalAmountController.test.ts (hunk 4 above)
Comment (GorilaNaranja) [reply src/test/unit/routes/features/eligibility/totalAmountController.test.ts]: Done!

File: src/test/unit/routes/features/eligibility/totalAmountController.test.ts [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+    it('should render not eligible page when radio less-25000 is selected', async () => {
+      await request(app).post(ELIGIBILITY_CLAIM_VALUE_URL).send({ 'totalAmount': 'less-25000' }).expect((res) => {
+        expect(res.status).toBe(302);
+        expect(res.header.location).toBe(ELIGIBILITY_SINGLE_DEFENDANT_URL);
+      });
+    });
+
+    it('should render not eligible page when radio unknow is selected', async () => {
+      await request(app).post(ELIGIBILITY_CLAIM_VALUE_URL).send({ 'totalAmount': 'unknow' }).expect((res) => {
```
[the remark this answers is not in the document]
Comment (GorilaNaranja) [reply src/test/unit/routes/features/eligibility/totalAmountController.test.ts]: Done!

**Pull request merged**